st/clitests/aws_iam_credential_file

```diff
@@ -1,5 +1,3 @@
 [default]
-aws_access_key_id = G6OFkXkYTyS_YRmxenLbBg
-aws_secret_access_key = KPykpvZqBQFysZ8vkmGwrPLXyafnOJ52n/qpRVOF
-
-
+aws_access_key_id = AKIAZhILo7aqSKWuT_YLVtutaQ
```

Comment:
this change should not be part of this PR

Reply:
oh...yes...